packages/mui-material/src/styles/experimental_extendTheme.d.ts

```diff
@@ -64,6 +64,10 @@ export type Overlays = [
   string | undefined,
 ];
 
+export interface PaletteBackgroundChannel {
+  defaultChannel: string;
+}
```

Comment:
For curiosity, why does `palette.background.default` has been removed?

Reply:
It is not removed but extended. Take a look at the changes in `packages/mui-material/src/themeCssVarsAugmentation/index.d.ts`.

```js
interface TypeBackground extends PaletteBackgroundChannel {}
```

Does this answer your question?

Reply:
Yes, I'm always a bit confused with TS extends